sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/epkversion/LeaseStoreManagerImpl.java

```diff
@@ -500,6 +507,32 @@ private Flux<ServiceItemLeaseV1> listDocuments(String prefix) {
             .map(ServiceItemLeaseV1::fromDocument);
     }
 
+    private Flux<ServiceItemLeaseV1> listDocuments(String prefix, int top) {
+        if (prefix == null || prefix.isEmpty()) {
```

Comment:
This is mostly duplicate code as with `listDocuments(String prefix)`. Can we re-use it? May be by making top as Integer and passing null in case of getting all leases.

Reply:
Fixed in next iteration.